src/guide/data-methods.md

```diff
@@ -1,5 +1,7 @@
 # Data Properties and Methods
 
+<VideoLesson href="https://vueschool.io/lessons/methods-in-vue-3?friend=vuejs" title="Learn how to use methods on Vue School">Learn how to work with data and methods with a free Vue School lesson</VideoLesson>
```

Comment:
Because this section covers both data and methods, it seems misleading to have this lesson here at the top of the doc.